src/Microsoft.AspNetCore.WebUtilities/HttpRequestStreamReader.cs

```diff
@@ -107,23 +108,20 @@ public HttpRequestStreamReader(Stream stream, Encoding encoding, int bufferSize)
                 }
             }
         }
-
+        
```

Comment:
(shameless plug) - install this: https://visualstudiogallery.msdn.microsoft.com/bcb3ef8e-a3a4-401e-9e63-fdf00b45268f/view/Reviews and you won't make this mistake again.